src/bilder/images/edxapp_v2/templates/edxapp/mitxonline/common_values.yml.tmpl

```diff
@@ -335,6 +335,24 @@ FINANCIAL_REPORTS:
     ROOT_PATH: sandbox
     STORAGE_TYPE: localfs
 FOOTER_ORGANIZATION_IMAGE: /images/logo.png
+FORUM_V2_ENABLE_MYSQL_BACKEND: true
+FORUM_SEARCH_BACKEND = "forum.search.es.ElasticsearchBackend"
+FORUM_ELASTIC_SEARCH_CONFIG = [
+    {
+        "host": "{{ keyOrDefault "elasticsearch/host" "elasticsearch.service.consul" }}"
+        "port": "443",
+        "use_ssl": True,
+    }
+]
+FORUM_ELASTIC_SEARCH_CONFIG_ES7 = [
+    {
+        "host": "{{ keyOrDefault "elasticsearch/host" "elasticsearch.service.consul" }}",
+        "port": "443",
+        "use_ssl": True,
+    }
+]
+FORUM_MONGODB_CLIENT_PARAMETERS = mongo_params
+FORUM_MONGODB_DATABASE = "edxapp"
```

Comment:
This needs to be YAML formatted. Right now it's using Python syntax.

Reply:
Thank you I should have run yamllint!